spec/carrierwave/cloudflare/responsive_images_helper_spec.rb

```diff
@@ -52,7 +52,15 @@ class DummyView
 
       it "generates image tag with srcset" do
         img_tag = view.hidpi_image_tag("/img.jpg", width: 400, dprs: [1, 2, 3])
-        expected_img_tag = "<img srcset=\"/cdn-cgi/image/width=400,dpr=1/img.jpg 1x, /cdn-cgi/image/width=400,dpr=2/img.jpg 2x, /cdn-cgi/image/width=400,dpr=3/img.jpg 3x\" src=\"/cdn-cgi/image/width=400/img.jpg\" />"
+        expected_img_tag = "<img srcset=\"/cdn-cgi/image/width=400,dpr=1/img.jpg 1x, /cdn-cgi/image/width=400,dpr=2/img.jpg 2x, /cdn-cgi/image/width=400,dpr=3/img.jpg 3x\" width=\"400\" src=\"/cdn-cgi/image/width=400/img.jpg\" />"
+
+        expect(img_tag).to eql(expected_img_tag)
+      end
+
+      it "generates image tag with width and height" do
+        img_tag = view.hidpi_image_tag("/img.jpg", width: 400, height: 300, dprs: [1, 2, 3])
+
+        expected_img_tag = "<img srcset=\"/cdn-cgi/image/width=400,height=300,dpr=1/img.jpg 1x, /cdn-cgi/image/width=400,height=300,dpr=2/img.jpg 2x, /cdn-cgi/image/width=400,height=300,dpr=3/img.jpg 3x\" width=\"400\" height=\"300\" src=\"/cdn-cgi/image/width=400,height=300/img.jpg\" />"
```

Comment:
Is it possible to break this line to maintain the editor width?

Reply:
Done